cli/internal/commands/run/command.go

```diff
@@ -204,7 +234,12 @@ func (o *Options) listStormForgeTestCaseNames() tea.Msg {
 		}
 		for j := range testCases.Data {
 			testCase := testCases.Data[j].Attributes.Name
-			msg = append(msg, fmt.Sprintf("%s/%s", org, testCase))
+			if len(orgs.Data) == 1 {
```

Comment:
nit, I wonder if it might be a good practice to always fully qualify the test case ( org/testcase )? Or does it cause issues with a single org user?